torch/ao/quantization/experimental/quantizer.py

```diff
@@ -47,28 +33,41 @@ def __init__(
     def quantize_APoT(self, tensor2quantize: Tensor):
```

Comment:
can you add type annotation for output as well?

Reply:
@jerryzh168 i didn't include type annotations for the return value (TensorAPoT) because when I do I get a circular import error (quantizer imports APoT_Tensor and APoT_Tensor imports quantizer). is it necessary to specify return value type? if so i can try to figure out another solution

Reply:
note: after discussion decided to just leave comment explaining APoT Tensor type